index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <figure id="img-div">
+          <img
+            id="image"
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+            alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+          />
+          <figcaption id="img-caption">
+            Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+            on how to increase wheat yields - part of his life-long war on hunger.
+          </figcaption>
+        </figure>
+        <section id="tribute-info">
+          <h3 id="headline">Here's a time line of Dr. Borlaug's life:</h3>
+          <ul>
```

Comment:
Please add class to your ul tag so you can handle your ul in CSS after this is your container you need add class here and its child li tag as well